tests/msc3030_test.go

```diff
@@ -63,6 +63,45 @@ func TestJumpToDateEndpoint(t *testing.T) {
 			mustCheckEventisReturnedForTime(t, alice, roomID, eventB.AfterTimestamp, "f", "")
 		})
 
+		t.Run("should find next event topologically before given timestmap when all message timestamps are the same", func(t *testing.T) {
```

Comment:
small typo:
```suggestion
		t.Run("should find next event topologically before given timestamp when all message timestamps are the same", func(t *testing.T) {
```

and the same for the other test too